README.md

```diff
@@ -6,7 +6,7 @@ Cubing Competition Manager (CCM) [![Build Status](https://travis-ci.org/cubing/c
 
 ### Optional setup (useful for development)
 - Install nodejs.
-- `(cd tests; npm install)` - Install linting tools and setup pre-commit hook.
+- `(cd tests; npm install --save-dev)` - Install linting tools and setup pre-commit hook.
```

Comment:
So I'm pretty sure --save-dev is only for when installing new packages, not for when installing packages from package.json. See <http://stackoverflow.com/a/19223182/1739415>.

I just tried this out locally with a fresh clone, and it worked for me, so I'm not sure what was happening for you:

```bash
~/gitting/ccm @kaladin> cd /tmp
/tmp @kaladin> git clone git@github.com:cubing/ccm.git
Cloning into 'ccm'...
cdremote: Counting objects: 6937, done.
remote: Compressing objects: 100% (12/12), done.
Receiving objects:  47% (3261/6937), 1.89 MiB | 1.81 MiB/s     
remote: Total 6937 (delta 5), reused 0 (delta 0), pack-reused 6925
Receiving objects: 100% (6937/6937), 3.18 MiB | 1.81 MiB/s, done.
Resolving deltas: 100% (4919/4919), done.
Checking connectivity... done.
/tmp @kaladin> cd ccm
/tmp/ccm @kaladin> (cd tests; npm install)
npm WARN prefer global jsonlint@1.6.2 should be installed with -g

> git-validate@2.1.4 install /tmp/ccm/tests/node_modules/git-validate
> node bin/install


> precommit-hook@3.0.0 install /tmp/ccm/tests/node_modules/precommit-hook
> node bin/install

ccm@0.0.0 /tmp/ccm/tests
├─┬ chai@3.5.0 
...
/tmp/ccm @kaladin> (cd tests; npm run-script lint)

> ccm@0.0.0 lint /tmp/ccm/tests
> (jshint --exclude-path ../.jshintignore --config ../.jshintrc .. && jscs ..)

/tmp/ccm @kaladin> 
```